tasks.py

```diff
@@ -63,3 +64,40 @@ def process_send_notifications():
         insert_q2 = construct_mail_sent_query(SYSTEM_EMAIL_GRAPH, bestuurseenheid_graph, bericht['bericht']['value'], email['uuid'])
         # helpers.log(insert_q2)
         update(insert_q2)
+
+        
+def send_confirmation_notifications():
+    """
+    Check for messages in berichtencentrum received from kalliope and send email to dossierbehandelaar letting them know the email has arrived succesfully
+
+    :returns ?:
+    """
+    helpers.log("fetching messages that need a confirmation notification to be sent ...")
+    q = find_kalliope_mail(PUBLIC_GRAPH, 70)
+    berichten = query(q)['results']['bindings']
+    helpers.log("found {} berichten. Processing ...".format(len(berichten)))
+    if berichten: # Prepare email template
+      here = os.path.dirname(os.path.realpath(__file__)) 
+      templatepath = os.path.join(here, 'templates/confirmatie_bericht.hbs')
+      with open(templatepath, 'r') as email_template_file:
+          compiler = Compiler()
+          precompiled = compiler.precompile(email_template_file.read())
+          email_html_template = compiler.template(precompiled)
+    for bericht in berichten:
+        subject = "Confirmatie {0}: '{1}' - Uw bericht is succesvol toegekomen in het berichtencentrum van {2}  ".format(bericht['dossiernummer']['value'], bericht['betreft']['value'], bericht['ontvanger']['value'])
+        content = None # NOTE: Not used, we use html_content
+        email = new_email("noreply@test.test", bericht['mailadres']['value'], subject, content) 
```

Comment:
Same here, the address should be configurable no ?

Reply:
Yes this should be the noreply email similar to regular notifications so replaced that by the correct ENV